src/thor/route_action.cc

```diff
@@ -192,7 +192,11 @@ std::string thor_worker_t::expansion(Api& request) {
   }
 
   // track the expansion
-  route(request);
+  try {
+    route(request);
+  } catch (...) {
+    // we swallow exceptions because we actually want to see what the heck the expansion did anyway
```

Comment:
this was a nice addition to make the `/expansion` endpoint more useful. you can now "debug" failed routes